src/lib/public/board/types.ts

```diff
@@ -53,7 +54,8 @@ interface Position {
 
 export interface Officer {
   fullName: string;
-  picture: string;
+  picture?: string;
+  legacyPicture?: string;
```

Comment:
Please leave a JSDoc comment explaining the `{#if}` mechanism that conditionally renders the officer picture.